code/controllers/subsystem/explosions.dm

```diff
@@ -108,8 +108,9 @@ SUBSYSTEM_DEF(explosions)
 	var/max_range = max(devastation_range, heavy_impact_range, light_impact_range, flame_range, throw_range)
 	var/started_at = REALTIMEOFDAY
 	if(adminlog)
-		message_admins("Explosion with size ([devastation_range], [heavy_impact_range], [light_impact_range], [flame_range]) in [ADMIN_VERBOSEJMP(epicenter)]")
 		log_game("Explosion with size ([devastation_range], [heavy_impact_range], [light_impact_range], [flame_range]) in [loc_name(epicenter)]")
+		if(is_mainship_level(epicenter.z))
```

Comment:
Why showing them during hijack?

Reply:
Because I want to avoid the gap between hijack depart-hijack crash

Reply:
Then why not adding another state for monitor, or on ticker?

Reply:
because i dont want to add more monitor states for something that is already an improvment in a 2 line pr